punishments.js

```diff
@@ -46,8 +54,8 @@ Punishments.hosts = new Map();
 // punishments.tsv is in the format:
 // punishType, userid, ips/usernames, expiration time
 
-Punishments.loadPunishments = function () {
-	fs.readFile(PUNISHMENT_FILE, (err, data) => {
+function punishmentLoader(filename, useridMap, ipMap) {
```

Comment:
Didn't I tell you not to make function names a noun?

Also why isn't this exported?

Also `useridMap`, `ipMap` don't need to be passed